src/ArmTemplates/Creator/TemplateCreators/PolicyTemplateCreator.cs

```diff
@@ -77,24 +77,24 @@ public PolicyTemplateResource CreateAPIPolicyTemplateResource(APIConfig api, str
 
         public PolicyTemplateResource CreateProductPolicyTemplateResource(ProductConfig product, string[] dependsOn)
         {
-            if (string.IsNullOrEmpty(product.name))
+            if (string.IsNullOrEmpty(product.Name))
```

Comment:
```suggestion
            if (string.IsNullOrWhitespace(product.Name))
```

Reply:
Do not want to modify `Creator` right now. Changes there were made only due to renaming fixes, that is why I left the comment in PR description

```
Please, do not comment Creator files - I didn't even start refactoring that code, so there are some rename fixes only. Did not change logic at all.
```